openff/toolkit/tests/test_forcefield.py

```diff
@@ -3782,8 +3796,6 @@ def test_freesolv_gbsa_energies(
 
         off_nonbonded_force.setReactionFieldDielectric(1.0)
 
-        # TODO: look into if switching distance is relevant #882
-
```

Comment:
It does - unsurprisingly. I'm not interested in documenting it but several tests have failed energy comparisons  locally. When the switching was applied in the `System` created by the toolkit but not in the one that went through Amber files, the energies differed.